docs/bot-usage.md

```diff
@@ -36,14 +36,16 @@ optional arguments:
 ```
 usage: freqtrade trade [-h] [-v] [--logfile FILE] [-V] [-c PATH] [-d PATH]
                        [--userdir PATH] [-s NAME] [--strategy-path PATH]
-                       [--db-url PATH] [--sd-notify]
+                       [--db-url PATH] [--sd-notify] [--dry-run]
 
 optional arguments:
   -h, --help            show this help message and exit
   --db-url PATH         Override trades database URL, this is useful in custom
                         deployments (default: `sqlite:///tradesv3.sqlite` for
                         Live Run mode, `sqlite://` for Dry Run).
   --sd-notify           Notify systemd service manager.
+  --dry-run             Enforce dry-run for trading, removes API keys and
```

Comment:
Maybe:
"Enforce dry-run for trading (it removes API key, secret and password and simulates trades)."
?

or "(this option removes...)"

or simply "Enforce dry-run for trading: remove API keys and simulate trades."

Reply:
how about now ... ?

Reply:
LGTM 👍